scripts/lib/mkcloud-common.sh

```diff
@@ -10,6 +10,7 @@
 : ${distsuse:=dist.nue.suse.com}
 distsuseip=$(dig -t A +short $distsuse)
 : ${susedownload:=download.nue.suse.com}
+: ${smturl:="$susedownload/update/build.suse.de"}
```

Comment:
why not make it a proper URL, starting with http:// ? because you are adding it everywhere anyway